anoncreds/src/data_types/anoncreds/master_secret.rs

```diff
@@ -14,7 +14,7 @@ impl MasterSecret {
     #[inline]
     pub fn new() -> Result<Self, ConversionError> {
         let value = UrsaProver::new_master_secret().map_err(|err| {
-            ConversionError::from_msg(format!("Error creating master secret: {}", err))
+            ConversionError::from_msg(format!("Error creating master secret: {err}"))
```

Comment:
NB this requires Rust 1.58 minimum. I think that's reasonable but should be added to Cargo.toml. 2021 edition requires 1.56 minimum so we can update to that as well.

Reply:
Yeah I can add 2021 to this PR.